src/main/java/org/jitsi/utils/dsi/DominantSpeakerIdentification.java

```diff
@@ -18,9 +18,11 @@
 import java.lang.ref.*;
 import java.util.*;
 import java.util.concurrent.*;
+import java.util.logging.Level;
```

Comment:
Change those to package imports.

Reply:
the change you suggested causes a compile error.
> both class org.jitsi.utils.logging.Logger in org.jitsi.utils.logging and class java.util.logging.Logger in java.util.logging match
any better resolution?

Reply:
Any reason you're using `logger.log(LEVEL.INFO...`, etc. as opposed to `logger.info(` ?

Reply:
removed.